includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,284 @@
+			'guides'                      => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'How to Sell Online Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Creating Video Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Choose the Right Hosting Provider',
+						'url'   => 'http://...',
+					],
+				],
+				'more_url' => 'http://senseilms.com/category/guides/',
+			],
+			'news'                        => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'Introducing Interactive Videos For WordPress',
+						'date'  => '2022-08-31', // Localized for user.
+						'url'   => 'http://senseilms.com/inroducing-interactive-videos/',
+					],
+					[
+						'title' => 'New Block Visibility, Scheduled Content, and Group Features',
+						'date'  => '2022-08-09', // Localized for user.
+						'url'   => 'http://senseilms.com/conditional-content/',
+					],
+				],
+				'more_url' => 'https://senseilms.com/blog/',
+			],
+			'extensions'                  => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro}.json.
+				[
+					'title'        => 'Sensei LMS Post to Course Creator',
+					'image'        => 'http://senseilms.com/wp-content/uploads/2022/02/sensei-post-to-course-80x80.png',
+					'description'  => 'Turn your blog posts into online courses.',
+					'price'        => 0,
+					'product_slug' => 'sensei-post-to-course', // To be used with the installation function `Sensei_Setup_Wizard::install_extensions`.
+					'more_url'     => 'http://senseilms.com/product/sensei-lms-post-to-course-creator/',
+				],
+			],
+			'show_sensei_pro_promo'       => false, // Whether we should show the promotional banner for Sensei Pro or not.
+			'notifications'               => [
+				[
+					'title'            => 'Your Sensei Pro license expires on 12.09.2022.',
+					'main_action'      => [
+						'title' => 'Update now',
+						'url'   => 'https://...',
+					],
+					'secondary_action' => [
+						'title' => 'What\'s new',
+						'url'   => 'https://...',
+					],
+					'level'            => 'error', // One of: info, warning, error.
+					'dismissible'      => false, // The default value is true.
+				],
+				[
+					'title'            => 'Good news, reminder to update to latest version',
+					'main_action'      => [
+						'title' => 'Update now',
+						'url'   => 'https://...',
+					],
+					'secondary_action' => [
+						'title' => 'Link for more information',
+						'url'   => 'https://...',
+					],
+					'level'            => 'info', // One of: info, warning, error.
+					'dismissible'      => true, // The default value is true.
+				],
+			],
+			'products_pending_activation' => [ 'sensei-pro' ],
```

Comment:
This part has been added to model the form that will allow for the activation of Sensei Pro license.

Reply:
While I know that this is only a mockup, I'm not sure about putting this data here, to be honest.

With that said, I might be wrong, but.. .do we need to know which products are pending activation?  I feel like a boolean or even an object containing keys like `activated`, `product_name` (if needed, but not sure), and `expiration_date` is already enough. 

I wonder if there's a situation where we might have more than one product pending activation..

Reply:
> While I know that this is only a mockup, I'm not sure about putting this data here, to be honest.

I don't understand this. Do you mean that you would rather have a separate endpoint?

> With that said, I might be wrong, but.. .do we need to know which products are pending activation? I feel like a boolean or even an object containing keys like activated, product_name (if needed, but not sure), and expiration_date is already enough.

As far as my understanding a single boolean wouldn't work because we might want to differentiate between sensei-pro and interactive-blocks at least. Maybe we can keep it like this and once we start implementing the actual licensing part decide better on this (to be honest haven't looked too much into it).

Reply:
> I don't understand this. Do you mean that you would rather have a separate endpoint?

Nah, it's only that I think this should be present on Sensei Pro instead, being added by a filter or something.

> As far as my understanding a single boolean wouldn't work because we might want to differentiate between sensei-pro and interactive-blocks at least. Maybe we can keep it like this and once we start implementing the actual licensing part decide better on this (to be honest haven't looked too much into it).

Fair enough.